libraries/azure-app-configuration-importer/tests/featureFlagConfigurationSettingsConverter.spec.ts

```diff
@@ -329,4 +329,174 @@ describe("Parse FeatureFlag Json format file", () => {
     );
     assert.isTrue(featureFlag3.enabled);
   });
+
```

Comment:
Could you add a testcase for old schema and new schema hybrid scenario?

Reply:
What should be the expected result? Currently if we have both `feature_management` and `FeatureManagement` an error will be thrown because there are multiple instances of feature management.